src/dataframe/io.jl

```diff
@@ -801,7 +801,7 @@ function readtable(io::IO,
                    separator::Char = ',',
                    quotemark::Vector{Char} = ['"'],
                    decimal::Char = '.',
-                   nastrings::Vector = ["", "NA"],
+                   nastrings::Vector = ["", "NULL", "NA"],
```

Comment:
Well, empty fields can also mean `""` (empty string) for `String` columns, so that cannot replace `NULL` in all cases. It's also a good idea to provide some flexibility here, e.g. to read files produced by R. And yes, we do write `NULL` for missing values by default.